demo/demoscript.js

```diff
@@ -0,0 +1,20 @@
+playground.button.then(function(x){

```

Comment:
This `then` is redundant and should be gone after you'll get rid of `import()` in entry point.